modules/Microsoft.Compute/disks/.test/image/deploy.test.bicep

```diff
@@ -23,7 +23,7 @@ param enableDefaultTelemetry bool = true
 
 // General resources
 // =================
-resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2022-09-01' = {
```

Comment:
Thanks for pointing this out and it definitely makes sense to me. I'm just wondering if it wouldn't be better to park API version update on dependency resources for now and manage that in bulk for all modules in a separated PR to avoid inconsistencies across the library. At least for the most used resources such as rg, storage, kv and vnet. What's your take on that?

Reply:
I think that it may be not be easy to keep the consistency across the library that way, for at least two reasons:
- I just searched in the library and there are already different API versions there for the popular resource types mentioned above
- while it's relatively easy to avoid changing API versions in existing modules, creation of new ones would require the contributors to explicitly use older API versions, which they probably don't even know

I agree that we should have a separate PR updating API versions for (at least) popular dependencies, but I think that (thank to the powerful Search/Replace function of VS Code) the complexity of this task will not significantly increase when we must search for different existing API versions. 

I'm happy to follow your recommendation, it's not a big deal to revert these updates.

Reply:
As agreed in the sync call, resolving the conversation.